### ISSUE TITLE
Add documentation for using the proxy within a phoenix router

### ISSUE THREAD

**Issue by CrowdHailer** [tommy351/plug-proxy#5]
I can't work out how to add this plug to a Phoenix router.

It looks like all the docs are for adding the plug to a plug router? is it possible to use this plug in phoenix?

**Comment by tejo** [comment]
yes you can: https://gist.github.com/tejo/c76c3b1ca3dd42f723ccff1c206cf988
